src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs

```diff
@@ -33,24 +33,14 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     public partial class GeneralOptionsDialogControl : UserControl
     {
         private readonly ISonarLintSettings settings;
-        private readonly ISonarLintDaemon daemon;
-        private readonly IDaemonInstaller installer;
         private readonly ILogger logger;
 
-        public GeneralOptionsDialogControl(ISonarLintSettings settings, ISonarLintDaemon daemon, IDaemonInstaller installer, ICommand openSettingsFileCommand, ILogger logger)
```

Comment:
This commit just deletes the installer and daemon classes. I'll fix up the options page and remove the unnecessary resources in subsequent PRs.